repository-3.0.json

```diff
@@ -1,5 +1,47 @@
 {
     "drivers": [
+        {
+        	"name": "chromedriver",
+        	"platform": "windows",
+        	"bit": "32",
+        	"version": "116.0.5845.96",
+        	"url": "https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.96/win32/chromedriver-win32.zip",
+        	"fileMatchInside": ".*chromedriver.exe$"
+        },
+        {
+        	"name": "chromedriver",
+        	"platform": "windows",
+        	"bit": "32",
```

Comment:
that should be "64", otherwise it looks good

Reply:
I'm sorry for this mistake, I commit a new version with this error corrected.